dist/APIManager.js

```diff
@@ -0,0 +1,39 @@
+//This is the class that will manage all your APIs
+
+class APIManager {
+
+    constructor() {
+        this.renderer = new Renderer()
```

Comment:
the renderer shouldn't be created and used here. this class should only manage the API calls, while the `main.js` file should be the one that using both APIManager and the Renderer classes